.github/workflows/ci.yml

```
@@ -19,10 +19,13 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Debug Working Directory
+        run: ls -la  # 디렉터리 상태 출력
+
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '20'
+          node-version: '20'  
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)